### ISSUE TITLE
Structure of Biocatalyst section not quite right

### ISSUE THREAD

**Issue by PeterHalling** [StephanM87/Strenda-biocatalysis#53]
I think the structure of this section needs some adjustment. We have sub-sections for Source and Formulation. In the case of a self-produced biocatalyst, Source covers description of the process used. Preparation of a biocatalyst up to the point it is placed in the reaction mixture will often involve multiple steps. In some cases it will be slightly arbitrary whether a particular step is classed as part of production or formulation. However, I think it is useful to keep these two sections. We should perhaps add an explanation of how we define these stages, e.g. “Production is taken to include all steps up to making what could be seen as a biocatalyst preparation, including separation from the production medium, washing, any purification applied, drying. The resulting preparation will often then be in a form suitable for storage, or transfer to another laboratory. Formulation covers all further steps that might be applied before the biocatalyst is placed in the reaction mixture. This includes addition of excipients, chemical modification, immobilisation, pre-equilibration (e.g. of water content). In many cases there will be no Formulation, the biocatalyst will be placed in the reaction mixture in the originally produced or purchased form.”

We then have a title “Biocatalyst Application” with sub-categories. The title is not very helpful (probably everything about an experiment would fall under Biocatalyst Application), and should be removed, I think. The sub-categories should then be redistributed under Source/Production or Formulation. For example, as Katrin pointed out in an earlier Issue, “Cell-free production” is clearly one type of production process. “Secreted Biocatalyst” is also a type of production. These belong as options under the section on "Production".

“Immobilisation” appears as separate heading. But I think it fits much better as a special case of formulation (as suggested above). 

In summary, I think it best that we have 3 main sub-headings: Source/Production, Formulation, Storage. Then appropriate lower level headings under each – the different types of production process, the different formulation options etc.

I would be happy to edit the whole section along these lines if wanted - but clearly other opinions need to be heard first.
